path_concise.json

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
For these JSON files that are only used in unit tests, should we pull them into a new `tests\data` directory?

Reply:
moved